lib/mozart_fetcher/components/ares.ex

```diff
@@ -0,0 +1,7 @@
+defmodule MozartFetcher.Components.Ares do
+  alias MozartFetcher.Components.Ares
+  @moduledoc false
+  @derive [Jason.Encoder]
```

Comment:
```suggestion
```
I don't think we need this anymore, as Jason doesn't support the option `as` like Poison did. https://github.com/michalmuskala/jason#differences-to-poison

Reply:
The link suggests that you DO have to derive the encoder if you want to encode structs so I'm a bit confused 🤔 

Good point about potentially not needing them to be structs at all though, I like it for the type safety but perhaps it's adding unnecessary complications

Reply:
whoops yeah sorry, I got muddled between encoding and decoding... :/